k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -252,6 +252,9 @@ function deployKubernetesPrereqs()
         createPodSecurityPolicy="false"
     fi
 
+    external_services_type=$(parseSolution 'solution.common.external_services.type')
+    external_services_type=$(echo $external_services_type | cut -f2 -d'>')
```

Comment:
What's the difference between `parseSolution` and `extractBlock`? Can you not simply use the latter and avoid the extra cut, like for the others below?

```
--set services.io.type="$(extractBlock 'solution.common.external_services.type' || true)" \
```

Reply:
Good catch. Based on other patterns here, I wasn't discerning between the two enough. Switched to run with extractBlock.